drivers/hmis/app/jobs/hmis/merge_clients_job.rb

```diff
@@ -16,7 +16,17 @@ def perform(client_ids:, actor_id:)
       raise 'You cannot merge less than two clients' if Array.wrap(client_ids).length < 2
 
       self.actor = User.find(actor_id)
-      self.clients = Hmis::Hud::Client.preload(:names, :contact_points, :addresses).order(Hmis::Hud::Client.arel_table['DateCreated']).find(client_ids)
+      self.clients = Hmis::Hud::Client.
+        preload(:names, :contact_points, :addresses).
+        find(client_ids).
+        map do |client|
+          # set some defaults
+          client.DateCreated ||= 10.years.ago.to_date
+          client.DateUpdated ||= 10.years.ago.to_date
+          client
+        end.
```

Comment:
This behavior mimics the client cleanup job. Clients with missing dates are assumed to be old, and will be chosen over more recent records. I'm not sure if that's right tbh, but favoring consistency with the wh

https://github.com/greenriver/hmis-warehouse/blob/5b08f19398508a2e96b258da7b9057e9cd71051d/app/models/grda_warehouse/tasks/client_cleanup.rb#L586-L587